rllib/algorithms/tests/test_algorithm_config.py

```diff
@@ -185,6 +189,225 @@ class A:
         config.validate()
         self.assertEqual(config.learner_class, A)
 
+    def _assertEqualMARLSpecs(self, spec1, spec2):
+        self.assertEqual(spec1.marl_module_class, spec2.marl_module_class)
+
+        self.assertEqual(set(spec1.module_specs.keys()), set(spec2.module_specs.keys()))
+        for k, module_spec1 in spec1.module_specs.items():
+            module_spec2 = spec2.module_specs[k]
+
+            self.assertEqual(module_spec1.module_class, module_spec2.module_class)
+            self.assertEqual(
+                module_spec1.observation_space, module_spec2.observation_space
+            )
+            self.assertEqual(module_spec1.action_space, module_spec2.action_space)
+            self.assertEqual(
```

Comment:
It is tested on non empty dicts.

Reply:
if you change this to a from `ray.rllib.utils.test_utils` import check call it will fix the problem. Then later if someone uses this utility then the assertion check will work

Reply:
oh I see ok